src/Microsoft.AspNet.Server.Kestrel/Http/Frame.cs

```diff
@@ -307,16 +307,19 @@ public void Abort()
             {
                 ConnectionControl.End(ProduceEndType.SocketDisconnect);
                 SocketInput.AbortAwaiting();
-                RequestAbortedSource.Cancel();
             }
             catch (Exception ex)
             {
                 Log.LogError("Abort", ex);
             }
-            finally
+
+            try
             {
-                _abortedCts = null;
+                RequestAbortedSource.Cancel();
```

Comment:
I don't think we should swallow this error. I think there is a chance that exceptions from callbacks registered with the token will bubble up through this call. I would probably use `KestrelTrace.ApplicationError`.